testing/oak_echo_service/proto/oak_echo.proto

```diff
@@ -26,7 +29,14 @@ message EchoResponse {
   bytes body = 1;
 }
 
+message GetEvidenceResponse {
+  oak.attestation.v1.Evidence dice_evidence = 1;
```

Comment:
I think we can just call it `evidence` at this point, since the old evidence has been removed

Reply:
Done, thanks.